.gitignore

```
@@ -1,29 +1,39 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
+node_modules
+dist
+dist-ssr
+*.local
 
-# editors
+# Editor directories and files
+.vscode/*
+!.vscode/extensions.json
```

Possiblblement ne pas partager les extensions vscode, tout le monde ne veut pas les mêmes.

```suggestion
.vscode
```